### PR TITLE
feat(settings): split usage indicators

### PR DESCRIPTION
**Pull request by jsfez**
Standard Plan
Capture d’écran 2022-09-23 à 11 17 46

Free Plan
Capture d’écran 2022-09-23 à 11 17 32

No Plan
Capture d’écran 2022-09-23 à 11 02 52

No Plan with open details
Capture d’écran 2022-09-23 à 11 02 57

### CODE REVIEW

File: apps/app/src/pages/Owner/GeneralSettings.js [hunk 1]
```diff
@@ -106,41 +119,143 @@ function PlanCard({ purchases, screenshotsLimitPerMonth, ...props }) {
...
     </Card>
   );
 }
 
 export function GeneralSettings({
-  owner: { name, purchases, currentMonthUsedScreenshots },
+  owner: { name, purchases, screenshotsLimitPerMonth, repositories },
 }) {
-  const screenshotsLimitPerMonth = Infinity;
+  const minScreenshotLimit = screenshotsLimitPerMonth || 7000;
```
Comment (gregberge) [inline apps/app/src/pages/Owner/GeneralSettings.js:258]: hmm ?

**Review by gregberge — CHANGES_REQUESTED** [review]

File: apps/app/src/pages/Owner/GeneralSettings.js (hunk 1 above)
Comment (jsfez) [reply apps/app/src/pages/Owner/GeneralSettings.js:258]: They are 2 interesting topic here :
- the ˋ|| 7000ˋ is here for people without plan purchase. By default repo have 7000 private screenshot / month even without plan. 

- the second topic is that I have here replaced `infinity` by a the plan's screenshots imite. It's maybe too early ?

Comment (gregberge) [reply apps/app/src/pages/Owner/GeneralSettings.js:258]: Actually it id unlimited for every one, so please remove this fake limit.

File: apps/app/src/pages/Owner/GeneralSettings.js [hunk 2]
```diff
@@ -42,7 +56,7 @@ const FeatureIcon = (props) => <Icon mt="-1px" {...props} />;
 const PlanName = (props) => <x.div fontSize="4xl" mb={3} {...props} />;
 
 const Plan = ({ purchase }) => {
-  if (!purchase.plan) return <PlanName mb={2}>No plan</PlanName>;
+  if (!purchase?.plan) return <PlanName mb={2}>No plan</PlanName>;
```
Comment (gregberge) [inline apps/app/src/pages/Owner/GeneralSettings.js:59]: Does not exist, we fallback on Free plan server-side.

**Review by gregberge — CHANGES_REQUESTED** [review]

File: apps/app/src/pages/Owner/GeneralSettings.js (hunk 2 above)
Comment (gregberge) [inline apps/app/src/pages/Owner/GeneralSettings.js:59]: No plan does not exist. It fallbacks on free plan. If not plan is defined in database then we have a special case where we say that there is no plan configured on this Argos instance.

**Review by gregberge — CHANGES_REQUESTED** [review]

**Pull request closed without merging**